### ISSUE TITLE
Error 500 when viewing a paused serie with annuals ('mylar.logger' has no attribute 'fdefbug')

### ISSUE THREAD

**Issue by mcarbenay** [mylar3/mylar3#1508]
Hi !

I'm currently using 0.7.8 on a docker configuration. I've added a serie and then paused the comic. When opening it again, I've got an 500.

Traceback (most recent call last):
  File "/lsiopy/lib/python3.11/site-packages/cherrypy/_cprequest.py", line 638, in respond
    self._do_respond(path_info)
  File "/lsiopy/lib/python3.11/site-packages/cherrypy/_cprequest.py", line 697, in _do_respond
    response.body = self.handler()
                    ^^^^^^^^^^^^^^
  File "/lsiopy/lib/python3.11/site-packages/cherrypy/lib/encoding.py", line 223, in __call__
    self.body = self.oldhandler(*args, **kwargs)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/lsiopy/lib/python3.11/site-packages/cherrypy/_cpdispatch.py", line 54, in __call__
    return self.callable(*self.args, **self.kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/mylar3/mylar/webserve.py", line 396, in comicDetails
    updater.forceRescan(ComicID)
  File "/app/mylar3/mylar/updater.py", line 1789, in forceRescan
    logger.fdefbug('[PAUSE_ANNUAL_CHECK_STATUS_CHECK] series is paused, keeping status of %s for issue #%s' % (issStatus, chk['Issue_Number']))
    ^^^^^^^^^^^^^^
AttributeError: module 'mylar.logger' has no attribute 'fdefbug'

 - OS: Linux Docker on TrueNas Scale
 - version 0.7.8

[carepackage_v0.7.8.zip](https://github.com/mylar3/mylar3/files/14237981/carepackage_v0.7.8.zip)

**Comment by evilhero** [comment]
Yeah 'tis a typo, but will still be a bit before we can merge the fix up into master unfortunately, since we never directly merge into master. 

In the meantime, just unpause the series and let it finish the check if required if the only reason why it's being paused is for testing.(Manage tab / Manage Comics /use the dropdown to unpause the given series). 

You can also switch to the nightly lsio image which is tied to the python3-dev branch of mylar, and the fix will be available there later today (when I get time to push it up to dev).

**Comment by mcarbenay** [comment]
Hi !

If it's already fixed/on the backlog : great :)

Thanx for the suggested work around.
